### DISCUSSION TITLE
Palworld startup arguments

### DISCUSSION THREAD

**Discussion by VenomDrake** [GameServerManagers/LinuxGSM#4459] [Q&A]
Hi everyone, I just finished configuring the Palworld server, but my only problem is that it doesn't appear in the server list.
I read that you need to add EpicApp=PalServer
but I don't understand where I should add it.
Does anyone know how and where?

Thank you

**Comment by elliotks** [comment]
That is included already in: /data/config-lgsm/pwserver/_default.cfg
https://github.com/GameServerManagers/LinuxGSM/blob/4e72b082c61905617665a35db40628dfb4e011e5/lgsm/config-default/config-lgsm/pwserver/_default.cfg#L19

**Reply by VenomDrake** [reply]
True, but the server isn't shown in the list anyway.
What did I do wrong?
